zkvm-prover/print_plonky3gpu_version.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+config_file=~/.cargo/config.toml
+plonky3_gpu_path=$(grep 'path.*plonky3-gpu' "$config_file" | cut -d'"' -f2 | head -n 1)
+plonky3_gpu_path=$(dirname "$plonky3_gpu_path")
+
+if [ -z $plonky3_gpu_path ]; then
+    exit 0
+else
+    pushd $plonky3_gpu_path
```

Shellcheck (0.10.0)

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)